book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

```diff
@@ -99,22 +99,28 @@ This will not go into much detail about about how to use Docker -- for more deta
     b. "Initialize" the data for the PEcAn database.
 
         ```bash
-        docker-compose run --rm bety initialize
+        docker run --rm --network pecan_pecan pecan/db
         ```
         
         This should produce a lot of output describing the database operations happening under the hood.
-        Some of these will look like errors (including starting with `ERROR`), but _this is normal_.
-        This command succeeded if the output ends with the following:
+<<<<<<< HEAD
```

Comment:
do you have a conflict?

Reply:
I thought I resolved this locally...didn't realize how it ended up here. Will fix it